src/rpc/blockchain.cpp

```diff
@@ -1526,7 +1535,14 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
     UniValue ret(UniValue::VOBJ);
     ret.push_back(Pair("time", (int64_t)pindex->nTime));
     ret.push_back(Pair("txcount", (int64_t)pindex->nChainTx));
-    ret.push_back(Pair("txrate", ((double)nTxDiff) / nTimeDiff));
+    if (blockcount > 0) {
+        ret.push_back(Pair("block_count", blockcount));
```

Comment:
Perhaps display `block_count` if it is equal to zero? (but not the other two values since they're meaningless if block_count == 0